samples/snippets/snippets_test.py

```diff
@@ -209,6 +217,14 @@ def test_download_blob(test_blob):
         assert dest_file.read()
 
 
+def test_download_blob_into_memory(test_blob):
+    contents = storage_download_into_memory.download_blob_into_memory(
+        test_blob.bucket.name, test_blob.name
+    )
+
+    assert contents == "Hello, is it me you're looking for?"
```

Comment:
Very good. Got my code review soundtrack playing now
https://www.youtube.com/watch?v=UBYnT8JY7sE

Reply:
Oh, this test isn't passing.
your method doesn't return a value, it prints it out, and so you need to capture the `out`

Like so:

```
    out, _ = capsys.readouterr()
    assert "Hello world" in out
    ```

Reply:
For reference:
https://github.com/googleapis/python-storage/pull/645/files

Reply:
Ah thank you!